pkg/chartutil/testdata/subpop/charts/subchart1/templates/service.yaml

```diff
@@ -5,9 +5,9 @@ metadata:
   labels:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release-name: "{{ .Release.Name }}"
-    kube-version/major: "{{ .Capabilities.KubeVersion.Major }}"
-    kube-version/minor: "{{ .Capabilities.KubeVersion.Minor }}"
-    kube-version/gitversion: "v{{ .Capabilities.KubeVersion.Major }}.{{ .Capabilities.KubeVersion.Minor }}.0"
+    kube-version/major: "{{ .Capabilities.Major }}"
+    kube-version/minor: "{{ .Capabilities.Minor }}"
+    kube-version/gitversion: "v{{ .Capabilities.Major }}.{{ .Capabilities.Minor }}.0"
```

Comment:
If this isn't really the `gitversion` should we even include the label?

Reply:
this is just testdata but I updated it for you @mattfarina